README.md

```diff
@@ -79,7 +79,8 @@ have several additional dependencies:
   ```
   # npm install -g grunt-cli yuidocjs grover bower gulp
   ```
-* Install [phantomjs](http://phantomjs.org)
+* Install [phantomjs](http://phantomjs.org) NB: Install the 1.9 branch, 2.0
```

Comment:
I would just phrase like:
```md
Install [phantomjs](http://phantomjs.org) version 1.9.x
```

also this kind of change should go in a separate commit